src/main/java/restaurant/entity/orders/Orders.java

```diff
@@ -0,0 +1,65 @@
+package restaurant.entity.orders;
+
+import java.util.Date;
+import java.util.List;
+
+import restaurant.entity.menu.Menu;
+import restaurant.entity.personal.Waiter;
+
+import javax.persistence.*;
+
+@Entity
+public class Orders {
+
+	@Id
+	private String id;
+
+	@Transient
+	private Waiter waiter;
+
+	@Column
+	private Integer table;
+
+	@Column
+	private Double totalPrice;
+
+	@Column
+	private Date placedAt;
+
+//	@ManyToMany(mappedBy = "order",fetch = FetchType.EAGER)
+	@Transient
+	private List<Menu> menuItems;
+
+	public Orders() {}
+
+	public Orders(Integer table, Waiter waiter, List<Menu> menuItems) {
+		this.table = table;
+		this.waiter = waiter;
+		this.menuItems = menuItems;
+	}
+
+	public String getId(){return id;}
+
+	public Integer getTable() {return table;}
+
+	public void setTable(Integer table) {this.table = table;}
+
+//	@ManyToOne(cascade = CascadeType.ALL, targetEntity = Waiter.class)
+//	@JoinColumn(name="waiter_id")
```

Comment:
De ce avem nevoie adnotările astea la metodă?

Reply:
Pentru waiter zicea tufisi ca nu e ok sa avem doar id-ul celuilalt obiect (gen idWaiter)
Dar nu poti avea evident nici coloana de tip Waiter
Cam asa am gasit ca se face
Pentru ca sa mearga ok decomentata asta trebuie corresponding label si in clasa Waiter 
Dupa ce se decomenteaza asta se poate sterge label-ul de @Transient de la waiter

Reply:
Asta da, dar ăsta e un simplu getter, nu înțeleg de ce ai adnota metoda de get și  nu câmpul de waiter în sine.

Reply:
https://www.baeldung.com/hibernate-one-to-many